main.nf

```diff
@@ -1798,9 +1859,47 @@ process BCFTOOLS_QUAST {
         $features \\
         --threads $task.cpus \\
         ${consensus.join(' ')}
+    ln -s quast/report.tsv
     """
 }
 
+////////////////////////////////////////////////////
+/* --            INTERSECT VARIANTS            -- */
+////////////////////////////////////////////////////
+
+/*
+ * STEP 5.8: Intersect variants with BCFTools
+ */
+if (!params.skip_variants && callers.size() > 2) {
```

Comment:
can't that be done with a `when` statement in the process?

Reply:
I always try to implement this sort of code with a `when` statement first and if things start failing tests because the channels arent created/initialised properly when factoring in various command-line options then I go for an `if` statement because its the safer option.